content_scripts/vimium_frontend.coffee

```diff
@@ -764,7 +757,7 @@ selectFoundInputElement = ->
   # heuristic of checking that the last anchor node is an ancestor of our element.
   if (findModeQueryHasResults && document.activeElement &&
       DomUtils.isSelectable(document.activeElement) &&
-      isDOMDescendant(findModeAnchorNode, document.activeElement))
+      DomUtils.isDescendant(findModeAnchorNode, document.activeElement))
```

Comment:
Suggest revert name to `isDOMDescendant`, unless there's a good reason to change it (?).